ambry-server/src/main/java/com/github/ambry/server/ParticipantsConsistencyChecker.java

```diff
@@ -39,22 +39,29 @@ public ParticipantsConsistencyChecker(List<ClusterParticipant> participants, Ser
 
   @Override
   public void run() {
-    // when code reaches here, it means there are at least two participants.
-    ClusterParticipant clusterParticipant = participants.get(0);
-    Set<String> sealedReplicas1 = new HashSet<>(clusterParticipant.getSealedReplicas());
-    Set<String> stoppedReplicas1 = new HashSet<>(clusterParticipant.getStoppedReplicas());
-    for (int i = 1; i < participants.size(); ++i) {
-      Set<String> sealedReplicas2 = new HashSet<>(participants.get(i).getSealedReplicas());
-      if (!sealedReplicas1.equals(sealedReplicas2)) {
-        logger.warn("Mismatch in sealed replicas. Set {} is different from set {}", sealedReplicas1, sealedReplicas2);
-        metrics.sealedReplicasMismatchCount.inc();
-      }
-      Set<String> stoppedReplicas2 = new HashSet<>(participants.get(i).getStoppedReplicas());
-      if (!stoppedReplicas1.equals(stoppedReplicas2)) {
-        logger.warn("Mismatch in stopped replicas. Set {} is different from set {}", stoppedReplicas1,
-            stoppedReplicas2);
-        metrics.stoppedReplicasMismatchCount.inc();
+    logger.info("Participant consistency checker is initiated. Participants count = {}", participants.size());
+    try {
+      // when code reaches here, it means there are at least two participants.
+      ClusterParticipant clusterParticipant = participants.get(0);
+      Set<String> sealedReplicas1 = new HashSet<>(clusterParticipant.getSealedReplicas());
+      Set<String> stoppedReplicas1 = new HashSet<>(clusterParticipant.getStoppedReplicas());
+      for (int i = 1; i < participants.size(); ++i) {
+        logger.info("Checking sealed replica list");
```

Comment:
do we need these info logs?

Reply:
I added to debug where the exception was thrown when running this checker. Will change it to DEBUG.